### ISSUE TITLE
footer section not working properly 

### ISSUE THREAD

**Issue by SALONI-AGGARWAL0304** [MonalikaPatnaik/UMatter#195]
I have noticed in footer section , while clicking on any of the option signup page is opening only , want to improve this , please assign me this issue under GSSOC'23

**Comment by Avinash-NCEIAN** [comment]
I want to work on this Project. Plzz assign it to me.

**Comment by MonalikaPatnaik** [comment]
Yeah..I know..Contact us and testimonials page is already assigned to someone..So, if you want to work on a particular button in footer section and redirect it there instead of sign-in page?

**Comment by Avinash-NCEIAN** [comment]
> Yeah..I know..Contact us and testimonials page is already assigned to someone..So, if you want to work on a particular button in footer section and redirect it there instead of sign-in page?

Yes

**Comment by ritik48** [comment]
Hey @MonalikaPatnaik ,
In the footer I would like to work on contact us page, please assign it to me if no one is working.

**Comment by SALONI-AGGARWAL0304** [comment]
hey @MonalikaPatnaik I would like to work upon directing it to the social sites pages as they are not working

**Comment by MonalikaPatnaik** [comment]
Contact Us page is already assigned...and there are no social sites as of now!
